core/src/test/scala/org/apache/spark/ui/UISeleniumSuite.scala

```diff
@@ -32,10 +32,8 @@ import org.apache.spark.api.java.StorageLevels
 import org.apache.spark.shuffle.FetchFailedException
 
 /**
- * Selenium tests for the Spark Web UI.  These tests are not run by default
- * because they're slow.
+ * Selenium tests for the Spark Web UI.
  */
-@DoNotDiscover
 class UISeleniumSuite extends FunSuite with WebBrowser with Matchers {
   implicit val webDriver: WebDriver = new HtmlUnitDriver
```

Comment:
We need to call `quit()` on this WebDriver in order to avoid leaking resources.  I'll change this to use `BeforeAndAfterAll`.